Mawdac/github-actions-sandbox
# new changelog testing

- don't run on push
- trigger the other workflows again
- push the changes
- update trigger
- debug options
- specify git refs
- specify tag
- try some valid tags
- definitely valid
- hell with the tags
- add custom config
- test changelog config location
- move it
- move it again
- check it out dummy
- try updating path first
- remove debug stuff
- 1 more debug
- start from 7.0.0
- quote it
- add the fricken v
- try this changelog ci action
- better naming
- update name and don't run it
- delete this recursive workflow
